vardbg/output/video_writer/config.py

```diff
@@ -58,18 +61,27 @@ def parse_hex_color(string):
 
     return r, g, b, 255
 
+# Some color_scheme can have same txt and bg color
+def color_contrast(body_txt):
```

Comment:
That sounds like a color scheme loading problem. No real color scheme will have the same foreground and background colors — how are people supposed to read the text if that's the case?